float32 gaz
float32[4] Kp
float32[4] Ki
float32[4] Kd
float32[4] Kp_vit
float32[4] Ki_vit
float32[4] Kd_vit
float32[4] consigne
float32[3] altitude_capteur
float32[2] altitude_vit
float32 temps_proc
float32[6] attitude
float32[6] accelMag
float32[4] erreur
float32[4] somme_erreurs
float32[4] variation_erreur
float32[4] erreur_vit
float32[4] somme_erreurs_vit
float32[4] variation_erreur_vit
float32 commande_devant_droit
float32 commande_devant_gauche
float32 commande_derierre_droit
float32 commande_derierre_gauche
float32 init_lacet
float32 frequence_capteur